ArcGISRuntimeSDKQt_CppSamples/Features/FeatureLayerQuery/FeatureLayerQuery.cpp

```diff
@@ -70,7 +70,7 @@ void FeatureLayerQuery::componentComplete()
     m_mapView->setMap(m_map);
 
     // create the feature table
-    m_featureTable = new ServiceFeatureTable(QUrl("https://sampleserver6.arcgisonline.com/arcgis/rest/services/USA/MapServer/2"), this);
+    m_featureTable = new ServiceFeatureTable(QUrl("https://services.arcgis.com/jIL9msH9OI208GCb/arcgis/rest/services/USA_Daytime_Population_2016/FeatureServer/0"), this);
```

Comment:
do screenshots need to be updated?

Reply:
@ldanzinger in this case the screenshots still look the same with the updated data